runtime-template-client/runtime-template-client.go

```diff
@@ -0,0 +1,197 @@
+package runtimetemplateclient
```

Comment:
update the file name to `runtime_template_client.go`, I realised that we might not have followed it diligently, but generally underscore are preferred over hyphen (no a rule set in stone, but almost all the files in the project follows that as now)